launchprocess.q

```diff
@@ -0,0 +1,45 @@
+/function which execute system commands from q, and is tailored to deal with the exit codes of lproc.sh:
+/cmd is a string and params are the process parameters which are passed to launch() 
+syscomm:{[params;cmd]
+
+    prevexitcode:first `$system cmd,"; echo $?";
```

Comment:
Should cast to an int instead of a symbol given the return code will always be numeric

```suggestion
    prevexitcode:first "I"$system cmd,"; echo $?";
```

Reply:
This wasn't working for me originally (because I wasn't using a capital I), that's why I tried using syms instead. Will do